### PR TITLE
new file organization

### LINKED ISSUE #21: Project folders can be refactored
The src folder has some components files together. I suggest putting those files in a folder just for components and each one in a specific directory.

### PR DESCRIPTION
**Pull request by Larissagilliane**
This PR is solving a problem related at issue #21 

Closes #21

### CODE REVIEW

File: src/components/App/App.jsx [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-import './App.css';
-import 'materialize-css/dist/css/materialize.min.css';
-const uuidv1 = require('uuid/v4');
+import React, { useState, useEffect } from "react";
+import BmiForm from "../BmiForm/BmiForm";
+import Info from "../Info/Info";
+import Bar from "../Bar/Bar";
+import "./App.css";
+import "materialize-css/dist/css/materialize.min.css";
+const uuidv1 = require("uuid/v4");
```
Comment (GermaVinsmoke) [inline src/components/App/App.jsx]: Can you replace all these double quotes with single quotes?

Comment (Larissagilliane) [reply src/components/App/App.jsx]: They're not in the same folder anymore

Comment (Larissagilliane) [reply src/components/App/App.jsx]: ohh sorry, i just got it wrong

**Comment by GermaVinsmoke** [comment]
Thanks for replacing those.

**Pull request merged**